letsencrypt/client/apache/configurator.py

```diff
@@ -75,27 +77,27 @@ class ApacheConfigurator(augeas_configurator.AugeasConfigurator):
     """
     zope.interface.implements(interfaces.IAuthenticator, interfaces.IInstaller)
 
-    def __init__(self, server_root=CONFIG.SERVER_ROOT, direc=None,
-                 ssl_options=CONFIG.OPTIONS_SSL_CONF, version=None):
+    def __init__(self, config, direc=None, version=None):
```

Comment:
If we are going to make config an object... it seems like it fits the singleton pattern pretty well.  It also seems like a bit of a nuisance to pass around the reference to the object to every function that needs it.